legacy/c3_trajectory_generator/src/waypoint_validity.cpp

```diff
@@ -0,0 +1,61 @@
+#include "waypoint_validity.hpp"
+
+// Point and sub_size must be relative to ogrid, IE: meters * 1/OGRID_RESOLUTION + offset
```

Comment:
The units here are not clear. Ogrid resolution is a tuple: `(rows x cols)`. What does it mean to have a unit of `meters` * `unitless ordered pair`?

Maybe you meant that it should be in terms of the size of a single grid in the ogrid.